src/public/log/schema.rs

```diff
@@ -1060,7 +1063,12 @@ mod test {
         policy_ids.insert(PolicyId::from_str("policy2").unwrap());
 
         let errors = (0..num_of_error)
-            .map(|i| ("error".to_string() + i.to_string().as_str()))
+            .map(|i| AuthorizationError::PolicyEvaluationError {
+                id: PolicyID::from_string(format!("policy{i}")),
+                error: EvaluationError::from(ExtensionFunctionLookupError::FuncDoesNotExist {
+                    name: Name::parse_unqualified_name("foo").unwrap(),
+                }),
+            })
```

Comment:
It's no longer possible to create an `AuthorizationError` from a string so I just created some nonsense errors to appease the tests (which don't appear to care what's in the errors).

Reply:
That's ok. I will create an issue to check the contents of these errors in one of the test scenarios.